CHANGELOG.rst

```diff
@@ -12,18 +12,28 @@ Internal Changes
 
 * **Gateway**: Add endpoint for refreshing expiring Gitlab tokens
 * **Sessions**: Check LFS size and available disk space before cloning
+* **Knowledge Graph**: Improve access tokens management
+* **Knowledge Graph**: Add migration importing Composite Plans into the Triples Store
+* **Knowledge Graph**: Prevent from data corruption caused by duplicate Project creation dates
 
 **Bug fixes**
 
 * **Sessions**: Handle expiring Gitlab tokens
 * **Gateway**: Do not remove Redis clients on logout
 * **Gateway**: Address security vulnerabilities
+* **Knowledge Graph**: Fix vulnerabilities in Graph Services' docker images
```

Comment:
Which docker images and which vulnerabilities? Do other images have these vuln also?

Reply:
Snyk was flagging up Graph Services' images only. The problems were:
```
✗ Critical severity vulnerability found in zlib/zlib
  Description: Out-of-bounds Write
  Info: https://security.snyk.io/vuln/SNYK-ALPINE314-ZLIB-2976174
  Introduced through: zlib/zlib@1.2.11-r3, apk-tools/apk-tools@2.12.5-r1, curl/libcurl@7.79.1-r3, curl/curl@7.79.1-r3
  From: zlib/zlib@1.2.11-r3
  From: apk-tools/apk-tools@2.12.5-r1 > zlib/zlib@1.2.11-r3
  From: curl/libcurl@7.79.1-r3 > zlib/zlib@1.2.11-r3
  and 1 more...
  Image layer: 'apk add --no-cache tzdata curl bash tini'
  Fixed in: 1.2.12-r2

✗ Critical severity vulnerability found in openssl/libcrypto1.1
  Description: Buffer Overflow
  Info: https://security.snyk.io/vuln/SNYK-ALPINE314-OPENSSL-1569445
  Introduced through: openssl/libcrypto1.1@1.1.1k-r0, openssl/libssl1.1@1.1.1k-r0, apk-tools/apk-tools@2.12.5-r1, libretls/libretls@3.3.3-r0, ca-certificates/ca-certificates@20191127-r5, curl/libcurl@7.79.1-r3
  From: openssl/libcrypto1.1@1.1.1k-r0
  From: openssl/libssl1.1@1.1.1k-r0 > openssl/libcrypto1.1@1.1.1k-r0
  From: apk-tools/apk-tools@2.12.5-r1 > openssl/libcrypto1.1@1.1.1k-r0
  and 7 more...
  Image layer: 'apk add --no-cache tzdata curl bash tini'
  Fixed in: 1.1.1l-r0

✗ Critical severity vulnerability found in apk-tools/apk-tools
  Description: Out-of-bounds Read
  Info: https://security.snyk.io/vuln/SNYK-ALPINE314-APKTOOLS-1533752
  Introduced through: apk-tools/apk-tools@2.12.5-r1
  From: apk-tools/apk-tools@2.12.5-r1
  Image layer: Introduced by your base image (openjdk:17-jdk-alpine3.14)
  Fixed in: 2.12.6-r0```

Reply:
And the fix was just to do version bumps?

How about: `Version bumps on some packages to address docker image vulnerabilities identified by snyk`.

Reply:
Actually, we switched to a different base image that is based on the latest Alpine. What about:
```
Switched to the latest Alpine linux to address docker image vulnerabilities identified by snyk
```
?